src/main/clm_varcon.F90

```diff
@@ -120,11 +120,12 @@ module clm_varcon
 
   real(r8), public :: thk_bedrock = 3.0_r8  ! thermal conductivity of 'typical' saturated granitic rock 
                                     ! (Clauser and Huenges, 1995)(W/m/K)
-  real(r8), public :: csol_bedrock = 2.0e6_r8 ! vol. heat capacity of granite/sandstone  J/(m3 K)(Shabbir, 2000) !scs
```

Comment:
Remove the !scs comment

Reply:
I think the '!scs' is actually on master?  History fields that could be activated are 'T_STEM' and 'DHSDT_CANOPY'.  I would not activate the new fields in FrictionVelocityMod.  I don't think there are restrictions for the namelist variables.

Reply:
OK, this comment is taken care of

[x] scs comment
[x] Added T_STEM and DHSDT_CANOPY to history when bhs is turned on.
[x] New history fields are default off